TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficCaptureSourceFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.opensearch.migrations.replay;
+
+import lombok.extern.slf4j.Slf4j;
+import org.opensearch.migrations.replay.kafka.KafkaBehavioralPolicy;
+import org.opensearch.migrations.replay.kafka.KafkaProtobufConsumer;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+
+@Slf4j
+public class TrafficCaptureSourceFactory {
+
+    public static ITrafficCaptureSource createTrafficCaptureSource(TrafficReplayer.Parameters appParams) throws IOException {
```

Comment:
nit: Since this takes the Parameters object & Parameters is a construct of the command line main class, this could e folded to within TrafficReplayer too.  It helps that it's really short.  I can't imagine another use case where I'd want this functionality other than from the command line where multiple options were supplied.

Reply:
I probably agree with you right now, my thought here is that when more sources are added in the future, it would be nice to contain this factory source logic here